src/ext.js

```diff
@@ -260,6 +261,7 @@ export default function ext({ translator, automationsEnabled }) {
               component: 'switch',
               translation: 'properties.enableToggle',
               ref: 'useEnabledCondition',
+              defaultValue: false,
```

Comment:
Is there a risk that these default values have changed now that they are explicitly set?

Reply:
well yes, if I did it incorrectly. I can double-check, maybe even do some manual sanity checks